K2Bridge.Tests.End2End/flights/MSearch_Viz_ExtendedStats.json

```diff
@@ -0,0 +1,50 @@
+{
+    "aggs": {
```

Comment:
this scenario is when there is no bucket?

Reply:
I think so